### PR TITLE
fix: CI failing due to PR #1724

### PR DESCRIPTION
**Pull request by armandmcqueen**
## Description

PR #1724 isn't passing CI after being merged to master. This is presumably due to new CI steps and a Go version change after that PR was opened. This PR fixes those issues. (1) apply autoformatter to code. (2) correct int -> string conversion bug that is flagged by new Go version.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

### CODE REVIEW

**Review by hamidzr — APPROVED** [review]
if this applies and is small could we add this in? https://github.com/determined-ai/determined/pull/1724/files#r560364629

**Comment by armandmcqueen** [comment]
I want to keep this PR just about fixing the CI failures, but I'll definitely make that change in the streaming API PR

File: master/internal/resourcemanagers/agent_resource_manager.go [hunk 1]
```diff
@@ -219,7 +220,7 @@ func (a *agentResourceManager) createResourcePoolSummary(
 			instanceTypeStringBuilder := strings.Builder{}
 			instanceTypeStringBuilder.WriteString(pool.Provider.GCP.InstanceType.MachineType)
 			instanceTypeStringBuilder.WriteString(", ")
-			instanceTypeStringBuilder.WriteString(string(pool.Provider.GCP.InstanceType.GPUNum))
+			instanceTypeStringBuilder.WriteString(fmt.Sprint(pool.Provider.GCP.InstanceType.GPUNum))
```
Comment (dzhu) [inline master/internal/resourcemanagers/agent_resource_manager.go:223]: `strconv.Itoa` would be a slightly better fit, I think. Although, looking at the context, it seems like this whole block ought to be replaced by a `fmt.Sprintf("%s, %dx%s", MachineType, GPUNum, GPUType)`.

Comment (armandmcqueen) [reply master/internal/resourcemanagers/agent_resource_manager.go]: Changed

**Review by dzhu — APPROVED** [review]

**Pull request merged**